src/libsync/configfile.cpp

```diff
@@ -170,6 +170,8 @@ ConfigFile::ConfigFile()
 
     QSettings settings(config, QSettings::IniFormat);
     settings.beginGroup(defaultConnection());
+
+    visible = true;
```

Comment:
This variable is set to true without any context; in which situation would it be set to false? Is it ever set to false? Should this not be read from or written to the config file?

Reply:
Yes, this is by-default "**true**" for **nextcloud** to keep same existing UI screen, no need to update it during runtime because all component should be always visible on nextcloud side, and we will set it de-fault "**false**" in our code for **magentacloud** customization.